modules/appeals_api/spec/requests/v2/higher_level_reviews_controller_spec.rb

```diff
@@ -143,12 +143,20 @@ def new_base_path(path)
       allow(client_stub).to receive(:upload).and_return(faraday_response)
       allow(faraday_response).to receive(:success?).and_return(true)
 
-      Sidekiq::Testing.inline! do
-        post(path, params: @data, headers: @headers)
-      end
+      with_settings(Settings.vanotify.services.lighthouse.template_id,
```

Comment:
This test is misnamed (and apparently has been for a while). The test says that it "create[s] the job to build the PDF" but this is not testing that, it's testing email notifications. Let's update the test description to be more accurate to what's being tested.

Reply:
on it